### PR TITLE
Regenerates Operator Example

### PR DESCRIPTION
**Pull request by danehans**
Regenerates the Operator example deployment due to https://github.com/projectcontour/contour-operator/pull/137.

Signed-off-by: Daneyon Hansen <daneyonhansen@gmail.com>

### CODE REVIEW

**Comment by skriss** [comment]
@danehans apologies for the churn, but I ended up reverting https://github.com/projectcontour/contour/pull/3159 since it was technically a breaking change to the API. I have a new draft PR up at https://github.com/projectcontour/contour/pull/3162 but going to wait for some additional reviews before merging it.

**Comment by danehans** [comment]
@skriss thanks for the heads-up. Commit `c3afd8e` updates the PR to only regenerate the operator example due to https://github.com/projectcontour/contour-operator/pull/137.

**Review by skriss — APPROVED** [review]
LGTM - just to confirm, no RBAC rules seem to be changing here, it's just being reordered (presumably alpha)

**Comment by danehans** [comment]
>no RBAC rules seem to be changing here, it's just being reordered 

@skriss that is correct.

**Pull request merged**
